dq-game/src/client/front_screen_handler.py

```diff
@@ -0,0 +1,9 @@
+from utils.socket_connection import FrontSocketConnection
+
+
+class FrontScreenHandler:
```

Comment:
Is this class really necessary ? 
We can just call from the main the `FrontSocketConnection`  and the listen. Or you will add more functionalities later? You can comment this class maybe with it's purpose

Reply:
it's not necessary but it's in my opinion a good application of the Dependency Inversion Principle. I want main.py to depend on an interface defining the interactions with the front screen and not with the details (like socket connection) directly